modules/database-commons/src/main/java/org/testcontainers/ext/ScriptUtils.java

```diff
@@ -185,6 +192,21 @@ else if (c == ' ' || c == '\n' || c == '\t') {
 		}
 	}
 
+	private static boolean contentMatches(char[] content, int offset, String needle) {
+		final char[] needleChars = needle.toCharArray();
+		final int end = offset + needleChars.length;
+		if (content.length < end) {
+			return false;
+		}
+
+		for (int i = 0; i < needleChars.length; i++) {
+			if (Character.toLowerCase(content[offset + i]) != Character.toLowerCase(needleChars[i])) {
+				return false;
+			}
+		}
+		return true;
+	}
```

Comment:
Performance probably isn't optimal; just aiming for clarity right now.
`needle` parameter is the substring we're looking for - I'll rename to be clearer.

Reply:
Changed